apps/builder/app/builder/features/topbar/domains.tsx

```diff
@@ -74,6 +85,39 @@ export const getStatus = (projectDomain: Domain) =>
     ? (`VERIFIED_${projectDomain.domain.status}` as const)
     : `UNVERIFIED`;
 
+export const PENDING_TIMEOUT = 60 * 3 * 1000;
```

Comment:
is it used somewhere else? otherwise remove export?

Reply:
yes used